README.md

```diff
@@ -65,12 +62,5 @@ To deploy a Tahoe Beta VM with Orka Desktop:
   ghcr.io/macstadium/orka-images/tahoe:latest
 ```
 
-To deploy a Tahoe Beta VM with SIP disabled using Orka Desktop:
```

Comment:
Are we not going to provide a SIP disabled image for Tahoe? The PR description does not mention anything about this, but we remove mention of it here